onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1004,6 +1004,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "Custom scale will be used if specified. Default value is 1/sqrt(head_size)",
               AttributeProto::FLOAT,
               OPTIONAL_VALUE)
+        .Attr("local_window_size",
```

Comment:
Shall we add back the causal attribute? Otherwise, we have to update document that this operator only support local attention or causal.

Reply:
I don't think we'll add it back until we have a use case for it... and for the above comment we also don't have a use case for adding both left and right window size attributes; I think it would be more confusing for users.

Reply:
That's fine to use one attribute.